src/d3d9/d3d9_swapchain.cpp

```diff
@@ -357,6 +358,7 @@ namespace dxvk {
       blitCreateInfo.mipLevels     = 1;
       blitCreateInfo.usage         = VK_IMAGE_USAGE_SAMPLED_BIT
                                    | VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT
+                                   | VK_IMAGE_USAGE_TRANSFER_DST_BIT
```

Comment:
Should be `SRC_BIT`, not a second `DST_BIT`, right?

Reply:
Oops, you're right of course.